src/shogun/preprocessor/PCA.cpp

```diff
@@ -102,168 +102,176 @@ bool CPCA::init(CFeatures* features)
 
 		// center data
 		Map<MatrixXd> fmatrix(feature_matrix.matrix, num_features, num_vectors);
+
```

Comment:
you can put your name to the authors at the beginning of the file
as well as changing the license to BSD (grep for BSD license in the source tree)